projects/igniteui-angular/src/lib/core/styles/components/grid/_grid-theme.scss

```diff
@@ -1504,6 +1504,19 @@
             height: rem(15px);
             min-width: rem(15px); /* yeah IE, it really needs to be 15px wide... */
             font-size: rem(15px);
+            position: relative;
+
+            &::after {
+                content: attr(data-sortIndex);
+                position: absolute;
+                top: -5px;
+                right: -1px;
+                font-size: rem(10px);
+                text-align: right;
+                font-family: sans-serif;
+                line-height: .625rem;
```

Comment:
@SisIvanova Please use the `rem` function so the resulting value takes the global `$browser-context` value into account.